### DISCUSSION TITLE
Shortest Path Plugin to Clue Scroll Plugin (Like Quest Helper)

### DISCUSSION THREAD

**Discussion by ghost** [runelite/runelite#18620] [Ideas]
Add a Shortest Path option to the Clue Scroll Plugin to quickly know how to get to your next clue step.
![clue scroll shortest path helper](https://github.com/user-attachments/assets/efa9c68e-aabb-4185-a1ce-9f016fae4060)

**Comment by mpickering** [comment]
You can use the "Shortest Clue" plugin - https://github.com/KeiranY/clue-pathing-runelite-plugin
